pyproject.toml

```diff
@@ -24,6 +24,7 @@ requires-python = ">=3.11"
 
 [project.scripts]
 phalanx = "phalanx.cli:main"
+update-shared-chart-version = "phalanx.control_system.update_shared_chart_version:run"
```

Comment:
Please add any new command-line functionality to the existing Phalanx CLI rather than adding a new CLI entry point (in this case, probably under `phalanx application`).

Reply:
I realized that this script is predicated on using the link mechanism for the shared charts rather than what is currently used. I'll have to redo it or just remove it.

Reply:
I've moved everything into the phalanx cli now.